package.json

```diff
@@ -4,7 +4,7 @@
   "version": "2.0.0-beta.3",
   "dependencies": {
     "@types/history": "^4.6.2",
-    "query-string": "6.0.0"
+    "qs": "6.5.1"
   },
   "peerDependencies": {
```

Comment:
"contributors": ["Matt Hawes <matt@mattie.mat> (https://github.com/MGHawes)"]

Reply:
Think I might change that email... :P